packages/cli/src/commands/link/linkDependency.js

```diff
@@ -51,9 +51,10 @@ const linkDependency = async (
 
     linkConfig.register(
       dependency.name,
-      dependency.config[platform] || {},
+      // $FlowExpectedError: We already checked if dependency.platforms[platform] exists
+      dependency.platforms[platform],
       params,
-      // $FlowFixMe: We check if project[platform] exists on line 42
+      // $FlowFixMe: We already checked if project[platform] exists
```

Comment:
got rid of these fixmes by extracting variables and asserting on that (pushed in 00abd83)